### PR TITLE
update parcel version & remove react lazy api

### PR DESCRIPTION
**Pull request by SirM2z**
Update parcel version to resolve build question.

The react lazy api is experimental, It has strange problems when combined with parcel's build command.

### CODE REVIEW

**Comment by SirM2z** [comment]
Need to wait for this pull request https://github.com/imgcook/pipboard/pull/15#issue-543299312

File: package.json [hunk 1]
```diff
@@ -56,7 +63,7 @@
     "express": "^4.17.1",
     "husky": "^4.3.6",
     "lint-staged": "^10.5.3",
-    "parcel-bundler": "^1.12.4",
+    "parcel": "^2.0.0-beta.1",
```
Comment (yorkie) [inline package.json:66]: Should we use beta version?

File: scripts/server.js [hunk 2]
```diff
@@ -39,7 +39,6 @@ app.get('/static/models/*', function(req, res) {
   }
 });
 
-const bundler = new Bundler('src/index.html');
-app.use(bundler.middleware());
-
-app.listen(Number(PORT));
+app.listen(Number(PORT), () => {
+  console.log('🤘 server is listeing on ' + PORT);
```
Comment (yorkie) [inline scripts/server.js]:
```suggestion
  console.log(`🤘 server is listening on ${PORT}`);
```

File: src/components/NewPipelineBox/index.jsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import React, { useState } from "react";
 import { Modal, Row, Col, Card, Tag, Button } from 'antd';
 
-import { PIPELINE_TEMPLATES } from '~/config/index';
-import { pipeline } from '~/common/service';
+import { PIPELINE_TEMPLATES } from 'src/config/index';
```
Comment (yorkie) [inline src/components/NewPipelineBox/index.jsx:4]: Why do we deprecate the usage of ~?

File: src/router/config.js [hunk 4]
```diff
@@ -1,26 +1,35 @@
...
+import Plugin from 'src/pages/Plugin';
+import Setting from 'src/pages/Setting';
+import Home from 'src/pages/Home';
 
 export default {
   "/tutorial": {
-    "/mnist": lazy(() => import('../pages/Tutorial/Mnist')),
-    "/assets-classification": lazy(() => import('../pages/Tutorial/AssetsClassification')),
-    "/": lazy(() => import('../pages/Tutorial'))
+    "/mnist": Mnist,
```
Comment (yorkie) [inline src/router/config.js]: Oops, we should use the single quotes here.

File: package.json (hunk 1 above)
Comment (SirM2z) [reply package.json:66]: Parcel 1 seems to have not been released for a long time.
The v2 is the default branch of parcel, and it alpha version released in 2019.9.

File: src/components/NewPipelineBox/index.jsx (hunk 3 above)
Comment (SirM2z) [reply src/components/NewPipelineBox/index.jsx:4]: The `~` is used by Parcel to resolve tilde paths(the nearest directory with package.json or the project root).

doc: https://v2.parceljs.org/features/module-resolution/#aliases

File: scripts/server.js (hunk 2 above)
Comment (SirM2z) [reply scripts/server.js]: Already updated

File: src/router/config.js (hunk 4 above)
Comment (SirM2z) [reply src/router/config.js]: Already updated

**Review by yorkie — APPROVED** [review]

**Pull request merged**